pyproject.toml

```diff
@@ -22,3 +22,6 @@ scikit-learn = "^1.4.0"
 [build-system]
 requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.isort]
+profile = "black"
```

Comment:
Just for my own knowledge: what does this do?

Reply:
We need to configure isort with black to specify that isort should use the import sorting rules that are recommended by black. This ensures that there are no discrepancies between the import code formatting enforced by isort and by black. 

Source: https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html